dlt-connector/src/graphql/enum/InputTransactionType.ts

```diff
@@ -0,0 +1,12 @@
+import { registerEnumType } from 'type-graphql'
+
+export enum InputTransactionType {
+  CREATION = 1,
+  SEND = 2,
+  RECEIVE = 3,
+}
+
+registerEnumType(InputTransactionType, {
+  name: 'InputTransactionType', // this one is mandatory
+  description: 'Type of the transaction', // this one is optional
+})

```

Comment:
CrossGroupType differ from InputTransactionType.
https://github.com/gradido/gradido_protocol/blob/master/proto/gradido/transaction_body.proto
LOCAL: Transactions on the same chain (transfer, creation, deferredTransfer, communityRoot, registerAddress)
INBOUND: Transactions entering blockchain from another chain (transfer - receiving part, registerAddress - move in)
OUTBOUND: Transactions leaving blockchain to another chain (transfer - sending part, registerAddress - move out)
CROSS: for cross group transaction which haven't a direction (group_friends_update)

For me they are complete different, so no merging.

Reply:
I have refactored the enums and add comments for them. I hope this make it more clear.
Now proto enums are in a sub folder from proto and graphql enums in a sub folder from graphql.
The protobuf emus are based on my proto files and in protobuf I need int values.
 
InputTransactionType describe a transaction type from backend.. send, receive or creation. 
CrossGroupType describe if the transaction goes from one community to another, or only on the same community/blockchain.  It say also the direction.. if transaction came to this blockchain (for example for send gradidos, INBOUND, the blockchain/community of recipient)
INBOUND -> blockchain
blockchain -> OUTBOUND
CrossGroupType is defined above the actually transaction type, so it can be used for SendGradido (GradidoTransfer) or RegisterAddress if someones moves his account from one community to another.